ajax/libs/html5-device-mockups/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "html5-device-mockups",
+  "filename": "dist/device-mockups.min.css",
+  "description": "HTML5 mockups of popular devices, to showcase your portfolio and spice up your website.",
+  "keywords": [
+    "html5",
+    "device",
+    "mockups",
+    "demo",
+    "portfolio"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/pixelsign/html5-device-mockups.git"
+  },
+  "homepage": "https://pixelsign.github.io/html5-device-mockups/",
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "https://github.com/pixelsign/html5-device-mockups.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "dist/**/*",
+          "device-mockups/**/!(*.psd)"
```

Comment:
`*.ai` should also be excluded here (`"device-mockups/**/!(*.psd|*.ai)"`), there are some ai files here:
```
./3.0.0/device-mockups/iPhone7/iPhone7_red.ai
./3.2.0/device-mockups/iPhone7/iPhone7_red.ai
./3.0.1/device-mockups/iPhone7/iPhone7_red.ai
./3.0.2/device-mockups/iPhone7/iPhone7_red.ai
./3.0.3/device-mockups/iPhone7/iPhone7_red.ai
./3.2.1/device-mockups/iPhone7/iPhone7_red.ai
./3.1.0/device-mockups/iPhone7/iPhone7_red.ai
./3.1.1/device-mockups/iPhone7/iPhone7_red.ai
```